### ISSUE TITLE
No manipulation of PV inside of bbappend

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#87]
a bbappend with the following should raise an error

```
PV = "1.2.3"
```

while 

```
PV ?= "1.2.3"
```
is okay
